webpack/assets/javascripts/react_app/components/common/table/formatters/hostsCountCellFormatter.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const hostsCountCellFormatter = controllerSingular => (
+  value,
+  { rowData: { name } },
+) => (
+  <a href={`hosts?search=${controllerSingular}+%3D+"${encodeURI(name)}"`}>
```

Comment:
is this supposed to be generic? I'm concerned it won't always be possible to generate the search query in this way, and perhaps we should provide it as a parameter, ref to another function etc? I'm fine with doing this in a later PR.

Reply:
Yes, it's supposed to be generic.